runtime/str.go

```diff
@@ -625,11 +625,20 @@ func strInterpolate(f *Frame, format string, values *Tuple) (*Object, *BaseExcep
 			if raised != nil {
 				return nil, raised
 			}
-			s, raised := ToStr(f, i)
-			if raised != nil {
-				return nil, raised
+			if matches[7] == "d" {
+				s, raised := ToStr(f, i)
+				if raised != nil {
+					return nil, raised
+				}
+				buf.WriteString(s.Value())
```

Comment:
Would be nice to have a single buf.WriteString() call to accompany valueIndex++. Maybe declare a `var val string` and then populate it in each conditional branch.